.github/workflows/buildAndTest.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - main
   pull_request:
+    branches:
+      - main
```

Comment:
I would leave it for all branches (or just ignore branches you don't want like LTC). Since we are starting to use llvm/torch-mlir branches as integration branches (or special test branches) let the CI run on them.